src/main/java/org/mqttbee/api/mqtt/mqtt3/message/publish/Mqtt3Publish.java

```diff
@@ -56,6 +56,12 @@ static Mqtt3PublishBuilder extend(@NotNull final Mqtt3Publish publish) {
     @NotNull
     Optional<ByteBuffer> getPayload();
 
+    /**
+     * @return the payload of this PUBLISH packet as byte array. Empty byte array if the payload is null.
+     */
+    @NotNull
+    byte[] getPayloadAsBytes();
```

Comment:
maybe getPayloadAsArray/getPayloadArray?
@christianhoff @david-katz @dobermai What do you think?

Reply:
I would stay with getPayloadAsBytes